### PR TITLE
first workflow file

### PR DESCRIPTION
**Pull request by mbianco**

### CODE REVIEW

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    container: mrtravis/gridtools:gcc-9-mpi
+    strategy:
+      build_type: [Debug]
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: Configure
+        run: |
+          mkdir -p build && cd build
+          cmake .. -DGHEX_BUILD_TESTS=ON
```
Comment (bettiolm) [inline .github/workflows/CI.yml]: No hint to GridTools path is required since everything is already set in `mrtravis/gridtools:gcc-9-mpi`, right?

Comment (mbianco) [reply .github/workflows/CI.yml]: That's my hope :)
I will try

**Comment by mbianco** [comment]
Seems to work now. Can we merge?

**Comment by mbianco** [comment]
Can I merge?

**Pull request merged**